thinkhazard_processing/scripts/initializedb.py

```diff
@@ -2,14 +2,29 @@
 from sqlalchemy import engine_from_config
 
 from ..models import Base
-from ..settings import settings
+from .. import settings
 
 
-def initdb(engine):
-    engine.execute("CREATE SCHEMA processing;")
+def initdb(engine, drop_all=False):
+    if drop_all:
+        Base.metadata.drop_all(engine)
+    if not schema_exists(engine, 'processing'):
+        engine.execute("CREATE SCHEMA processing IF NOT EXISTS;")
     Base.metadata.create_all(engine)
 
 
+def schema_exists(engine, schema_name):
+    connection = engine.connect()
+    sql = '''
+SELECT count(*) AS count
+FROM information_schema.schemata
+WHERE schema_name = '{}';
+'''.format(schema_name)
+    result = connection.execute(sql)
+    row = result.first()
+    return row[0] == 1
```

Comment:
What about using .scalar(). This would return the value directly if I'm not wrong.